services/ui-src/src/components/accordions/FaqAccordion.tsx

```diff
@@ -11,9 +11,7 @@ export const FaqAccordion = ({ accordionItems, ...props }: Props) => {
     <Accordion allowToggle={true} allowMultiple={true} {...props}>
       {accordionItems.map((item: AnyObject, index: number) => (
         <AccordionItem key={index} label={item.question} sx={sx.item}>
-          <Box sx={sx.answerBox}>
-            <Text>{parseCustomHtml(item.answer)}</Text>
-          </Box>
+          <Box sx={sx.answerBox}>{parseCustomHtml(item.answer)}</Box>
```

Comment:
Fixes `Warning: validateDOMNesting(...): <p> cannot appear as a descendant of <p>.`